noted/notes/v1/notes.proto

```diff
@@ -137,3 +144,12 @@ message DeleteBlockRequest {
 message DeleteBlockResponse {
 
 }
+
+message ExportNoteRequest {
+    string id = 1;
+    NoteExportFormat note_format = 2;
```

Comment:
This isn't the case in the notes-service but elsewhere we have tried to not use the name `id` too much because it's always kind of confusing (is it a group ID? a note ID? a block ID?). To disambiguate I suggest the following
```suggestion
    string note_id = 1;
    NoteExportFormat export_format = 2;
```